rasa/cli/export.py

```diff
@@ -66,9 +66,9 @@ def _get_tracker_store(endpoints: "AvailableEndpoints") -> "TrackerStore":
             f"Exiting. "
         )
 
-    from rasa.core.tracker_store import TrackerStore
+    from rasa.core.tracker_store import TrackerStore, AwaitableTrackerStore
 
-    return TrackerStore.create(endpoints.tracker_store)
+    return AwaitableTrackerStore.create(TrackerStore.create(endpoints.tracker_store))
```

Comment:
This will hide the use tracker store in the telemetry tracking [here](https://github.com/RasaHQ/rasa/blob/e09e1310754e6eb69441e7b4154c8d6d37cc7b16/rasa/telemetry.py#L806)